### ISSUE TITLE
Remember the last position of the cursor

### ISSUE THREAD

**Issue by ghitsh** [HappenApps/Quiver#282]
When reading multiple documents back and forth, quiver doesn't remember the last position of the cursor so I have to re-scroll the pages all over again and again. Is there any setting to jump to the last position of a note when opening it?

**Comment by ylian** [comment]
If you have to switch between multiple documents constantly, would it make more sense to open them in separate windows? The thing with restoring scroll position is that sometimes you want that, but other times you want the document to scroll to top.

**Comment by ghitsh** [comment]
Multiple documents meant to be multiple notes in Quiver. Say, I'm writing on a first note and try to copy some contents from a second note. I should click on the first note again to paste the contents after I copy them from the second note but the first note will point to the top of the note, which I have to scroll the pages to find where to paste. I think you could provide an option in Preferences to make users to decide.

**Comment by ylian** [comment]
Yeah, I meant multiple notes. You can open a note in a separate window by double clicking on it from the note list.

A global setting won't really help. For example, sometimes when I switch to a note, I want to see it scrolled to the last position, but other times, I prefer to see it scroll to the top.

**Comment by ylian** [comment]
I guess I can add a setting "Restore scroll position" but leave it off by default. If someone prefers to restore the scroll position at all times, he can turn it on.

**Comment by JMichaelTX** [comment]
> For example, sometimes when I switch to a note, I want to see it scrolled to the last position, but other times, I prefer to see it scroll to the top

This is why we need a shortcut key to take us to the top and bottom of the Note.
CMD+▲/▼ is bound by the cell rather than the Note.

So I also would like to open to the last position (if set in Prefs) if I could easily get to the top/bottom of the Note.

**Comment by ylian** [comment]
@JMichaelTX That makes sense. I will add a setting. Also if you have in mind what keyboard shortcut you like to use to scroll to top and bottom, let me know.

**Comment by JMichaelTX** [comment]
> Also if you have in mind what keyboard shortcut you like to use to scroll to top and bottom, let me know.

I'm good with any of these, whichever is the more intuitive and produces the fewer conflicts:
- CMD-OPT- ▲ | ▼
- CTRL- ▲ | ▼

But I'm very open to anything that works consistently and doesn't break any global shortcuts.
